app/Http/Controllers/Settings/SubscriptionController.php

```diff
@@ -93,6 +97,8 @@ public function change(Request $request, Tier $tier)
         }
         $upgrade = $this->subscriptionUpgrade->user($user)->tier($tier)->upgradePrice($period);
         $currency = $user->currencySymbol();
+        
+        $this->emailValidation->user($user)->requiresEmail();
```

Comment:
This is now sending the email to every single user opening the subscription page, while it should only send to people suspected of frauding _when_ they try and subscribe